app/controllers/sufia/uploads_controller.rb

```diff
@@ -73,6 +73,11 @@ def asset_attributes
         params.fetch(:generic_work, nil) || params.fetch(:batch_upload_item, nil)
       end
 
+      def duplicate_generic_work(fileset)
```

Comment:
can you refactor DuplicateUploadVerificationService to return the parent GenericWork instead of the FileSet?